### DISCUSSION TITLE
what am i doing wrong!

### DISCUSSION THREAD

**Discussion by murrayjadea** [nickpourazima/rotary-phone-audio-guestbook#53] [Q&A] [answered]
Everything works fine until turn it off and back on again. I'm new to all off this and have no idea what I am doing. Any suggestions? Please help! I've gotten so far:(

**Comment by nickpourazima** [comment, marked as answer]
My guess is that the systemctl service isn't active. Can you run this to check? 

`systemctl status audioGuestBook.service`

If you see something like `status: disabled` then you should issue the following:

`sudo systemctl enable audioGuestBook.service`

Otherwise, I would need some more context about the commands you've issued so far and your setup details.

**Reply by murrayjadea** [reply]
Thank you! I ended up figuring it out! I simply needed to reboot after I had made my changes. Now I have another issue though. After I record and hang up the phone it will randomly play the recording while on the hook. Its usually at random times. Like the last one I heard was 7 minues after I hung up the phone and then again after 4 mins. Any suggestions?

**Reply by nickpourazima** [reply]
Any chance you've tried the newest release? We've added a bunch of new features and fixed some bugs. This one in particular sounds very familiar.

**Comment by murrayjadea** [comment]
Ahh gezzz, yes! I had updated, it but I think it was the bounce time. It just needsd to be a little longer! Thank you

**Reply by nickpourazima** [reply]
Oh awesome! Glad you were able to figure it out!
